docs/usage.md

```diff
@@ -207,105 +255,326 @@ which are optional:
 - `node_groups`: HSM groups to add to the boot set
 - `rootfs_provider`: the root file system provider
 - `rootfs_provider_passthrough`: parameters to add to the `rootfs=` kernel
-    parameter
+  parameter
 
-The properties listed previously are the same as the parameters that can be
-specified directly through BOS boot sets. More information can be found in the
-[CSM documentation on session
-templates](https://cray-hpe.github.io/docs-csm/en-10/operations/boot_orchestration/session_templates/).
-Additional properties not listed are passed through to the BOS session template
-as written.
-  
-An example session template might look like the following:
+As mentioned above, the parameters under `bos_parameters` are passed through
+directly to BOS. Refer to the section **BOS Session Templates** of the [*Cray
+System Management Documentation*](https://cray-hpe.github.io/docs-csm/) for more
```

Comment:
This was actually intentionally not a link to the specific section because we don't want to have to maintain the `en-13` part of the link. There is some discussion about this (in the other direction, from docs-csm to docs-sat) here: https://github.com/Cray-HPE/docs-csm/pull/2366#discussion_r966317466

Jessica says it's the approach techpubs is taking for now for linking between different products' documentation.